symbolic-base/src/ZkFold/Symbolic/Data/Ed25519.hs

```diff
@@ -49,14 +44,14 @@ instance
     -- It will need additional checks in pointDouble because of the denominator becoming zero, though.
     -- TODO: Think of a better solution
     --
-    pieces Inf         = hliftA2 V.append <$> pieces (zero :: UInt 256 'Auto c) <*> pieces (zero :: UInt 256 'Auto c)
+    pieces Inf         = hliftA2 V.append <$> pieces (zero :: FFA Ed25519_Base c) <*> pieces (zero :: FFA Ed25519_Base c)
     pieces (Point x y) = hliftA2 V.append <$> pieces x <*> pieces y
 
     restore f = Point x y
         where
             (x, y) = restore f
 
-instance (Symbolic c, Eq (Bool c) (BaseField (Ed25519 c))) => Eq (Bool c) (Point (Ed25519 c)) where
+instance (Symbolic c) => Eq (Bool c) (Point (Ed25519 c)) where
```

Comment:
Does this actually work for circuits? I think we might get some unexpected errors when we use it in the circuit context. For example, `Inf /= Point zero zero` even though it's the same point.